MapboxCoreNavigation/RouteController.swift

```diff
@@ -325,7 +325,7 @@ open class RouteController: NSObject {
         var userCourse = calculatedCourseForLocationOnStep
         var userCoordinate = snappedCoordinate.coordinate
         
-        if location.course >= 0 || location.speed >= RouteControllerMinimumSpeedForLocationSnapping {
+        if location.course >= 0 && location.speed >= RouteControllerMinimumSpeedForLocationSnapping {
```

Comment:
Wouldn't this result in even less course snapping?

Reply:
Nope, this if statement is for when the user course/location is unsnapped.